l10n_us_check_writing_address/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2017 Ursa Information Systems <http://www.ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'US Check Printing with Payee Address',
+    'summary': 'Print US Checks',
+    'version': '11.0.0.1',
+    'license': 'AGPL-3',
+    'author': 'Ursa Information Systems, Odoo Community Association (OCA)',
+    'category': 'Localization/Checks Printing',
+    'maintainer': 'Ursa Information Systems',
+    'website': 'http://www.ursainfosystems.com',
+    'depends': [
+        'account_check_printing_report_base',
+    ],
+    'data': [
+    ],
+    'installable': True,
```

Comment:
@Sudarshan1607 Can you add:
```
    'development_status': 'Beta',
    'maintainers': ['max3903'],
```
If you add unit tests, you can put `Production/Stable` as the development status.